AdblockPlusSafari/AdblockPlusController.m

```diff
@@ -64,7 +66,7 @@ - (void)observeValueForKeyPath:(NSString *)keyPath ofObject:(id)object change:(N
 {
   if ([keyPath isEqualToString:NSStringFromSelector(@selector(enabled))]) {
     self.blockingEnablingSwitch.on = self.adblockPlus.enabled;
-    [self.tableView reloadSections:[NSIndexSet indexSetWithIndex:1] withRowAnimation:UITableViewRowAnimationNone];
+    [self.tableView reloadSections:[NSIndexSet indexSetWithIndexesInRange:NSMakeRange(0, 2)] withRowAnimation:UITableViewRowAnimationNone];
```

Comment:
Well this is basically Massive View Controller inception, so wrapping the index arithmetic away in a separate class would be good. The pluses and minuses should not be scattered across the controller. But this is already 50% better than the original hardcode, so let's do the other 50% when there is another UI requirement.